### PR TITLE
added support for multipass url generator

### PR DESCRIPTION
**Pull request by knopa**
I am not sure how better to do the test. Technically best option is to find the user in Shopify customers by multipass identifier to assert it.

### CODE REVIEW

**Comment by nozzlegear** [comment]
Thanks for the pull request @knopa! I don't think I'm familiar with this feature, can you explain what this is for, and add a link to some documentation?

**Comment by knopa** [comment]
It's from list unimplemented functionality 
https://help.shopify.com/api/reference/multipass

**Comment by knopa** [comment]
It's needed when you need to allow auto-create customers from your existing project not Shopify one

File: ShopifySharp/Services/Multipass/MultipassService.cs [hunk 1]
```diff
@@ -0,0 +1,99 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace ShopifySharp
+{
+    public class MultipassService
```
Comment (nozzlegear) [inline ShopifySharp/Services/Multipass/MultipassService.cs:10]: It looks like the only method on this class is `GetMultipassUrl`, would it make more sense to make the class static, remove the constructor, and then make the method take `myShopifyUrl` and `multipassSecret` in addition to `customer`?

**Review by nozzlegear — CHANGES_REQUESTED** [review]
Hey! I'm looking into finally merging this into master. I've left a review comment about changing the class to static, would you mind taking a look if you're still interested in getting this merged?

File: ShopifySharp/Services/Multipass/MultipassService.cs (hunk 1 above)
Comment (knopa) [reply ShopifySharp/Services/Multipass/MultipassService.cs]: Hey, I have updated as your asked.

**Comment by knopa** [comment]
Hey, I have updated as your asked.

**Review by nozzlegear — APPROVED** [review]

**Comment by nozzlegear** [comment]
Released in 5.18.0 on nuget! Thanks for your contribution!

**Pull request merged**